helm-chart/mezod/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v2
+name: mezod
+version: 0.0.1
+appVersion: v0.1.0-rc0
```

Comment:
We will need to bump the chart version on every release of `mezod`, correct? Is there a way to automate this process? E.g. a GH action that will periodically check the latest tag on `mezod` and commit the change here? Of course, if this is not a 1-hour thing, we don't have to do it now. Just curious.

Reply:
I don't know any tool automating this action, so we'll have to create our own scripts.